ManageWikiSettings.php

```diff
@@ -3886,6 +3895,25 @@
 		'help' => 'Whether or not to show the normal MediaWiki external link icon when using the Chameleon skin.',
 		'requires' => [],
 	],
+	'egUserProfileV2Color' => [
```

Comment:
should be `wg`, same for below.

Reply:
Changed, I forgot to turn the default values in `LocalSettings.php` to variables, that's why I use `eg` not `wg`